<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GtsStation_GtlCommandsdialog_base</class>
 <widget class="QDialog" name="GtsStation_GtlCommandsdialog_base">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>176</width>
    <height>317</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GTL commands</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupCommands">
     <property name="title">
      <string>GTL commands</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="buttonDebugON">
        <property name="text">
         <string>-gtl_debugon</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="buttonDebugOFF">
        <property name="text">
         <string>-gtl_debugoff</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="buttonQuietON">
        <property name="text">
         <string>-gtl_quieton</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="buttonQuietOFF">
        <property name="text">
         <string>-gtl_quietoff</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="buttonInfo">
        <property name="text">
         <string>-gtl_info</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="buttonStatus">
        <property name="text">
         <string>-gtl_status</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QRadioButton" name="buttonTest">
          <property name="text">
           <string>-gtl_test</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinTestNumber">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="maximum">
           <number>999999</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QRadioButton" name="buttonTestlist">
        <property name="text">
         <string>-gtl_testlist</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="buttonHelp">
        <property name="text">
         <string>-gtl_help</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="buttonNone">
        <property name="text">
         <string>NONE</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QPushButton" name="buttonSend">
       <property name="text">
        <string>&amp;Send</string>
       </property>
       <property name="shortcut">
        <string>Alt+R</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCancel">
       <property name="text">
        <string>&amp;Close</string>
       </property>
       <property name="shortcut">
        <string>Alt+C</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
